guardian/matrixstore-client
# Adding github build and improving versioning

## What does this change?

Provides a github build

## How to test

- PRs should auto-test
- `main` branch merges/pushes should build a prerelease and show it on the "releases" page. Name should end in -pre.jar
- tag pushes should build a release and show it on the releases page. Name should end in -{tag}.jar.
